src/Sylius/Bundle/ApiBundle/Resources/config/app/config.yaml

```diff
@@ -33,14 +33,21 @@ api_platform:
 framework:
     messenger:
         buses:
-            sylius_default.bus:
-                middleware:
-                    - 'validation'
-                    - 'doctrine_transaction'
-            sylius_event.bus:
-                middleware:
-                    - 'validation'
-                    - 'doctrine_transaction'
+            sylius.command_bus: &command_bus
+                default_middleware: allow_no_handlers
+            sylius.event_bus: &event_bus
+                default_middleware: allow_no_handlers
+            sylius_default.bus: *command_bus
+            sylius_event.bus: *event_bus
+            
+services:
```

Comment:
Why did you decide on an additional YAML services definition?

Reply:
that was only way to mark them as deprecated